### PR TITLE
Copy h2 'heading-large' style from GOVUK elements

### PR DESCRIPTION
**Pull request by katstevens**
Done as part of this ticket: https://trello.com/c/wxCSXbeH/159-update-homepage-group-framework-tasks-by-framework-and-add-headings

Until now we haven't needed the larger `h2` style - now we do (in the Admin home page). I've copied exactly what we've done for `heading-xmedium`.

### CODE REVIEW

File: toolkit/scss/shared_scss/_dmspeak.scss [hunk 1]
```diff
@@ -20,6 +20,10 @@
     @extend %heading-small;
   }
 
+  .heading-large {
```
Comment (TheDoubleK) [inline toolkit/scss/shared_scss/_dmspeak.scss]: I'd be tempted to order these small, medium, large in the file.  Or large, medium small.  But not medium, small, large.

File: toolkit/scss/shared_placeholders/_dm-typography.scss [hunk 2]
```diff
@@ -29,3 +29,15 @@
   }
 
 }
+
+// Cut and pasted from the `.heading-large` styles in the `_elements-typography.scss` file in GOVUK Elements:
```
Comment (TheDoubleK) [inline toolkit/scss/shared_placeholders/_dm-typography.scss]: Same here with ordering.

**Comment by samuelhwilliams** [comment]
I don't think I understand the need for this. Shouldn't all our headings at a given level be as 'important' as each other?

**Comment by katstevens** [comment]
The plan is to use `<h2 class="heading-xlarge">` and `<h3 class="heading-xmedium">`.

**Comment by katstevens** [comment]
> grammer

Sorry, this was a terrible Molesworth joke. I can squash 😸

File: toolkit/scss/shared_scss/_dmspeak.scss (hunk 1 above)
Comment (katstevens) [reply toolkit/scss/shared_scss/_dmspeak.scss]: Good spot - now fixed.

**Comment by samuelhwilliams** [comment]
Oh I see. It seems weird to me that we don't just define h1/h2/h3 directly as having certain styles... but I guess we don't, so this must be fine. 👍

**Comment by TheDoubleK** [comment]
Sorry for not getting the joke... it would have been fine to leave it in :)

There is a part of me that feels we should have basic styles defined for H1, H2, H3 too.

**Pull request merged**